pkg/common/config/config.go

```diff
@@ -945,6 +953,14 @@ func init() {
 	InitGCPViper()
 }
 
+func WriteSecret(name string) string {
+	tmpDir, _ := os.MkdirTemp("", "")
+	tmpFile, _ := os.CreateTemp(tmpDir, "*")
```

Comment:
```suggestion
	tmpFile, _ := os.CreateTemp("", "osde2e-userca-bundle-data-")
```
I think just writing to base tmp is fine but I'd give it a prefixed name